bin/disco_snapshot.py

```diff
@@ -89,7 +91,7 @@ def run():
 
     aws = DiscoAWS(config, environment_name=environment_name)
     if args.mode == "create":
-        aws.disco_storage.create_ebs_snapshot(args.hostclass, args.size)
+        aws.disco_storage.create_ebs_snapshot(args.hostclass, args.size, args.unencrypted)
```

Comment:
Shouldn't we negate `args.unencrypted` here? since the argument the function is expecting is `encrypted`